crates/services/sync/src/import.rs

```diff
@@ -623,16 +557,13 @@ async fn execute_and_commit<E>(
     executor: &E,
     state: &SharedMutex<State>,
     block: SealedBlock,
-) -> Result<(), ImportError>
+) -> anyhow::Result<()>
```

Comment:
I think it would be nice to move away from using `anyhow` inside internal implementations, but we don't have to solve that right now.